### PR TITLE
[11.0] [MIG] report_xlsx

### PR DESCRIPTION
**Pull request by etobella**
Migration to 11.0.
Creation of reports has changed a little because report_swx has disappeared on v11. However, changes are not complicated.

### CODE REVIEW

**Comment by gabrielcardoso21** [comment]
@etobella, very nice job! I was having really hard times migrating this module. I shure would get there, but it would take at least until the end of the week.

**Comment by pedrobaeza** [comment]
@gabrielcardoso21 please review then the PR.

@etobella any chance of increasing test coverage?

**Comment by etobella** [comment]
@pedrobaeza Test coverage improved, but controllers are not being tested right now.

**Review by rgarnau — APPROVED** [review]
LGTM :+1:

File: report_xlsx/README.rst [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed feedback.
+
+Credits
+=======
+
+Contributors
+------------
+
+* Adrien Peiffer <adrien.peiffer@acsone.eu>
```
Comment (elicoidal) [inline report_xlsx/README.rst:76]: I would suggest that you add your name as contributor

File: report_xlsx/__init__.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+# Copyright 2015 ACSONE SA/NV (<http://acsone.eu>)
```
Comment (elicoidal) [inline report_xlsx/__init__.py]: be consistent and remove the copyright notice in all init files

**Review by elicoidal — CHANGES_REQUESTED** [review]
some nitpicking.

File: report_xlsx/__init__.py (hunk 2 above)
Comment (pedrobaeza) [reply report_xlsx/__init__.py]: You can remove also `coding` line

Comment (etobella) [reply report_xlsx/__init__.py]: @elicoidal Suggested changes have been included.

Comment (etobella) [reply report_xlsx/__init__.py]: @pedrobaeza coding could be removed from all files, isn't it?

Comment (pedrobaeza) [reply report_xlsx/__init__.py]: Yeah, exactly. See second point of https://github.com/OCA/maintainer-tools/wiki/Migration-to-version-11.0#tasks-to-do-in-the-migration

**Review by JordiBForgeFlow — APPROVED** [review]

File: report_xlsx/tests/__init__.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
```
Comment (elicoidal) [inline report_xlsx/tests/__init__.py]: Some remaining.

**Review by elicoidal — APPROVED** [review]
js and xml can/should contain copyright notices too.
Not blocking for that though

**Pull request merged**